src/init/init.c

```diff
@@ -63,6 +67,52 @@ static inline void lp_sram_unpack(void)
 }
 #endif
 
+#if CONFIG_MULTICORE
+
+int slave_core_init(struct sof *sof)
```

Comment:
I feel like I remember that the reverse movement happened at some point, or do I remember wrong? I certainly can't recall the details.

I think bringing together the _generic_ init functions is a good thing.

Reply:
It may have, but there is nothing now in the slave init that is xtensa specific. Anything xtensa specific can be avstarcated later though for xtos but abstraction not needed here for Zephyr.

Reply:
Yeah, was explaining why I approved. And I meant generic in contrast to platform specific.